### PR TITLE
soc/power9: implement istep 13.4

### PR DESCRIPTION
**Pull request by krystian-hebel**
Signed-off-by: Krystian Hebel <krystian.hebel@3mdeb.com>

### CODE REVIEW

File: src/soc/ibm/power9/istep_13_4.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+		if (!mcs_functional[i])
+			continue;
+
+		// Drop PLDY bypass of Progdelay logic
+		/*
+		TP.TPCHIP.NET.PCBSLMC01.NET_CTRL1 (WAND)                  // 0x070F0045
+		  [all] 1
+		  [2]   CLK_PDLY_BYPASS_EN =  0
+		*/
+		write_scom_for_chiplet(mcs_ids[i], 0x070F0045, ~PPC_BIT(2));
```
Comment (miczyg1) [inline src/soc/ibm/power9/istep_13_4.c:55]: Preprocessor definitions of registers and bits

File: src/soc/ibm/power9/istep_13_4.c [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+		// Set scan ratio to 4:1
+		/*
+		TP.TCMC01.MCSLOW.OPCG_ALIGN                               // 0x07030001
+		  [47-51] SCAN_RATIO =  3           // 4:1
+		*/
+		scom_and_or_for_chiplet(mcs_ids[i], 0x07030001, ~PPC_BITMASK(47,51),
+		                        PPC_BIT(50) | PPC_BIT(51));
+
+		// > end if
```
Comment (miczyg1) [inline src/soc/ibm/power9/istep_13_4.c:116]: But there is no IF begginning

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/soc/ibm/power9/istep_13_4.c (hunk 2 above)
Comment (krystian-hebel) [reply src/soc/ibm/power9/istep_13_4.c:116]: Line 66

**Comment by macpijan** [comment]
Already integrated in: https://github.com/3mdeb/coreboot/pull/49

**Pull request closed without merging**